crud/unflatten_rows.go

```diff
@@ -0,0 +1,51 @@
+package crud
+
+import (
+	"context"
+
+	"github.com/tarantool/go-tarantool"
+)
+
+// UnflattenRowsRequest helps you to create request object to call
+// `crud.unflatten_rows` for execution by a Connection.
```

Comment:
`unflatten_rows` is a request to transform the output based on tuple returned info and schema returned info. And it's rather simple. It sounds bothersome to execute a network request to transform the output. We decided to reimplement in in tarantool-python (https://github.com/tarantool/tarantool-python/pull/264), and maybe you should do it here too.

(Let's discuss it with @oleg-jukovec , since this approach definitely has its downsides)

Reply:
Reworked.